src/processor/InstanceProcessor.ts

```diff
@@ -110,7 +110,13 @@ export class InstanceProcessor {
     const flatInstanceArray = toPairs(getPathValuePairs(inputJSON));
     flatInstanceArray.forEach(([path], i) => {
       const assignmentRule = new ExportableAssignmentRule(path);
-      assignmentRule.value = getFSHValue(i, flatInstanceArray, instanceOfJSON.type, fisher);
+      assignmentRule.value = getFSHValue(
+        i,
+        flatInstanceArray,
+        instanceOfJSON.type,
+        target.name,
```

Comment:
It looks like `target.name` is `{name}-of-{instanceOf}`, so the warning message says something like `MyObservation-of-Observation`, which is not super familiar to users and I don't think appears in the output FSH or input JSON. I didn't check enough to know if either of these are consistently available, but from my quick check, it looked like it might be possible to use `target.id` and `target.instanceOf` separately to get a little more readable version of the name in the warning, so the warning could have something like `MyObservation of Observation`.

Reply:
`id` definitely seems like the more useful information to give, since many Instances aren't going to have meaningful names. I'll update this to give a better reference.